src/test/java/cook/ParserTest.java

```diff
@@ -0,0 +1,206 @@
+package cook;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.HashMap;
+
+import org.junit.jupiter.api.Test;
```

Comment:
I think this import statement should be moved to line 4 (or line 3 to line 8), to maintain the import statement consistency. https://se-education.org/guides/conventions/java/intermediate.html#statements